pkg/webhookconfig/registration.go

```diff
@@ -220,28 +220,19 @@ func (wrc *Register) UpdateWebhookConfigurations(configHandler config.Interface)
 		<-wrc.UpdateWebhookChan
 		logger.V(4).Info("received the signal to update webhook configurations")
 
-		var nsSelector map[string]interface{}
+		// var nsSelector map[string]interface{}
 		webhookCfgs := configHandler.GetWebhooks()
-		if webhookCfgs != nil {
-			selector := webhookCfgs[0].NamespaceSelector
-			selectorBytes, err := json.Marshal(*selector)
-			if err != nil {
-				logger.Error(err, "failed to serialize namespaceSelector")
-				continue
-			}
-
-			if err = json.Unmarshal(selectorBytes, &nsSelector); err != nil {
-				logger.Error(err, "failed to convert namespaceSelector to the map")
-				continue
-			}
+		webhookCfg := config.WebhookConfig{}
+		if len(webhookCfgs) > 0 {
```

Comment:
Should probably error and `continue` if it's not `>0`

Reply:
I think providing no config at all is valid, no ?

Reply:
But i wonder why it's an array... What's the point of accepting an array and taking only the first item ?